src/sentry/utils/snuba.py

```diff
@@ -611,6 +611,8 @@ def snuba_query(params):
         try:
             body = json.loads(response.data)
         except ValueError:
+            if response.status != 200:
+                raise SnubaError("Failed to parse snuba error response")
```

Comment:
Will this scenario be captured in sentry? It seems like something we should be alerted on.